src/Accelerators/CMakeLists.txt

```diff
@@ -10,7 +11,8 @@ if (ONNX_MLIR_ACCELERATORS)
     add_subdirectory(${t})
 
     # If the accelerator can be built
-    if (${t}_ENABLED)
+    string(TOUPPER ${t} T)
```

Comment:
Just to normalize accelerator names to upper case, since all cmake variables seem to be in upper case.